### ISSUE TITLE
Lazy properties documentation includes unneeded code

### ISSUE THREAD

**Issue by ilyapuchka** [realm/jazzy#768]
When documenting lazy properties jazzy can include it's actual code as part of it's declaration documentation.

Example - https://github.com/krzysztofzablocki/Sourcery/blob/master/docs/Classes/TypesReflectionBox.html#L176 In this example `=` is included as part of declaration documentation. 

If the property body is defined in the same line it will be completely included in the declaration documentation. 

```swift 
public lazy var classes: [Class] = { return self.types.flatMap { $0 as? Class } }()
``` 

This code produces the following html output:

```html
<pre class="highlight"><code><span class="kd">public</span> <span class="kd">lazy</span> <span class="k">var</span> <span class="nv">classes</span><span class="p">:</span> <span class="p">[</span><span class="kt"><a href="../Classes/Class.html">Class</a></span><span class="p">]</span> <span class="o">=</span> <span class="p">{</span> <span class="k">return</span> <span class="k">self</span><span class="o">.</span><span class="n">types</span><span class="o">.</span><span class="n">flatMap</span> <span class="p">{</span> <span class="nv">$0</span> <span class="k">as?</span> <span class="kt"><a href="../Classes/Class.html">Class</a></span> <span class="p">}</span> <span class="p">}()</span></code></pre>
```

**Comment by reitzig** [comment]
Related: #776, #777

**Comment by johnfairh** [comment]
Fixed via #902.
